### ISSUE TITLE
Segmentation fault (core dumped) while trying to run tests in brand new project.

### ISSUE THREAD

**Issue by LNow** [lgalabru/clarinet#12]
`clarinet test` is failing with following error:
```
Segmentation fault (core dumped)
```
There are no additional information except above.

Steps to reproduce:
1. Install clarinet using: `$ cargo install clarinet`
2. Create new  project: `clarinet new my-project`
3. Create contract: `clarinet generate contract first-contract`
4. Run tests: `clarinet test`

Rust version: `rustc 1.51.0 (2fd73fabe 2021-03-23)`
Cargo version: `cargo 1.51.0 (43b129a20 2021-03-16)`

**Comment by lgalabru** [comment]
@LNow thank you for giving this project a shot!
Your error is interesting, that could be platform specific. What OS are you running?

**Comment by LNow** [comment]
I'm running:
```
System: Fedora release 33 (Thirty Three)
Kernel : 5.11.11-200.fc33.x86_64
```

I've tried to compile it from the source and it failed as well.
```sh
[lnow@fedora clarinet]$ cargo install --path .
  Installing clarinet v0.5.1 (/home/lnow/Projects/clarinet)
    Updating crates.io index
   Compiling typenum v1.13.0
   Compiling proc-macro-nested v0.1.7
   Compiling crunchy v0.2.2
   Compiling crc v1.8.1
   Compiling futures-util v0.3.14
   Compiling rusty_v8 v0.20.0
   Compiling rusty_v8 v0.21.0
   Compiling bip39 v1.0.1
   Compiling indexmap v1.6.2
   Compiling url v2.2.1
   Compiling wgpu-types v0.7.0
   Compiling serde_urlencoded v0.7.0
   Compiling ron v0.6.4
   Compiling generic-array v0.14.4
   Compiling tokio-util v0.6.5
   Compiling async-compression v0.3.7
error: failed to build archive: bad archive: truncated or malformed archive (offset to next archive member past the end of the archive after member elements.o)

error: aborting due to previous error

error: could not compile `rusty_v8`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to build archive: bad archive: truncated or malformed archive (offset to next archive member past the end of the archive after member elements.o)

error: aborting due to previous error

error: failed to build archive: bad archive: truncated or malformed archive (offset to next archive member past the end of the archive after member elements.o)

error: aborting due to previous error

error: failed to compile `clarinet v0.5.1 (/home/lnow/Projects/clarinet)`, intermediate artifacts can be found at `/home/lnow/Projects/clarinet/target`

Caused by:
  build failed
```
got the same error when I tried to run `cargo build --release`. But plain `cargo build` succeeded.

**Comment by lgalabru** [comment]
I see. I personally work on macOS, but I'll try to spend some cycles on a Linux box this week.
Will keep you posted!

**Comment by LNow** [comment]
I think this might be something related to how `cargo install` works.
Today I pulled new version of the code and `cargo build --release` succeeded without any problems. However `cargo install --path .` and `cargo install clarinet` still fails.

**Comment by lgalabru** [comment]
Addressed with `0.5.3`
